lib/chef/resource/windows_certificate.rb

```diff
@@ -207,16 +207,16 @@ def export_cert(cert_obj, cert_path)
           when ".der"
             out_file.puts(cert_obj.to_der)
           when ".cer"
-            cert_out = powershell_out("openssl x509 -text -inform DER -in #{cert_obj.to_pem} -outform CER").stdout
+            cert_out = powershell_exec("openssl x509 -text -inform DER -in #{cert_obj.to_pem} -outform CER").result
```

Comment:
What's the perf for powershell_exec vs. standard shellout because this seems like PS isn't needed

Reply:
we should use shell_out when calling narive commands because shell_out will make sure the path is properly set it is also faster since there is some overhead with powershell_exec. Locally I compared and there was ~300ms difference.